### PR TITLE
Accents

### PR DESCRIPTION
**Pull request by Pangogie**
Adds dog, cat, and a terribly implemented monkey accents.

:cl: Pancake
- add: Added accents for overly talkative animals.

### CODE REVIEW

File: Content.Server/Speech/EntitySystems/MonkeyAccentSystem.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                        else
+                            accentedMessage += "a";
+                    }
+                    
+                }
+
+                accentedMessage += " ";
+            }
+            accentedMessage = accentedMessage.Substring(0, accentedMessage.Length - 1);
+            accentedMessage.Substring(0, 1).Replace(accentedMessage[0], (accentedMessage[0].ToString().ToUpper()).ToCharArray()[0]);
```
Comment (PaulRitter) [inline Content.Server/Speech/EntitySystems/MonkeyAccentSystem.cs:56]: what are these two calls? afaict, they dont do anything

**Review by PaulRitter — CHANGES_REQUESTED** [review]
imo cat & dog should all be one system. do it like this

ReplacementAccentListPrototype for all the lists of replacements
ReplacementAccentComponent which holds the ReplacementAccentListPrototypeId
ReplacementAccentSystem that does the replacing

as for the monkey system: what is that based on or is that something you came up with yourself?

File: Content.Server/Speech/EntitySystems/MonkeyAccentSystem.cs (hunk 1 above)
Comment (metalgearsloth) [reply Content.Server/Speech/EntitySystems/MonkeyAccentSystem.cs]: Looks like it was just making the first letter uppercase?

**Review by PaulRitter — APPROVED** [review]
the two components should be ignored on client. besides that, lgtm

**Review by moonheart08 — APPROVED** [review]

**Comment by PaulRitter** [comment]
oh yeah forgot to merge this oops

**Pull request merged**